tests/TestCase/Cache/Engine/RedisEngineTest.php

```diff
@@ -192,13 +192,13 @@ public function testMultiDatabaseOperations(): void
      */
     public function testWriteNumbers(): void
     {
-        $result = Cache::write('test-counter', 1, 'redis');
+        Cache::write('test-counter', 1, 'redis');
```

Comment:
Wonder if we should always be validating the write return value is successful.

Reply:
Probably, though the subsequent read() call indirectly confirms that the write was successful.